Include/ast.h

```diff
@@ -4,6 +4,9 @@
 extern "C" {
 #endif
 
+#include "node.h"         /* node */
```

Comment:
Other option is to add forward declarations, as in `compile.h`.

Reply:
Well, maybe, but it seems like node.h is included anyway in all C files including ast.h. I don't see the benefit of avoiding to include node.h here. I prefer to be explicit and link ast.h to node.h.

Reply:
`#include "Python-ast.h"` is already surrounded with `extern "C" {` in ast.h.